doc/source/working_with_images/custom_partitions.rst

```diff
@@ -65,12 +65,17 @@ size="size_string"
   specification mega-Byte is used.
 
 mountpoint="path"
-  Mandatory mountpoint to mount the partition in the system.
+  Optional mountpoint to mount the partition in the system.
 
 filesystem="btrfs|ext2|ext3|ext4|squashfs|xfs
-  Mandatory filesystem configuration to create one of the supported
+  Optional filesystem configuration to create one of the supported
   filesystems on the partition.
 
+label="string"
+  Optional filesystem label if a filesystem is provided. If no
+  label is specified the `name` identifier is set as filesystem
+  label
+
```

Comment:
Is this still valid documentation?

Reply:
ah no, I overlooked that part, thanks for pointing it out

Reply:
yes it is. It's not directly related to the uuid and seed initiative but while implementing I found the documentation about the label handling was wrong and incomplete. I keep this change as an extra commit